plugins/listblock/plugin.js

```diff
@@ -101,7 +105,8 @@ CKEDITOR.plugins.add( 'listblock', {
 							'return false;" onmouseup="CKEDITOR.tools.getMouseButton(event)===CKEDITOR.MOUSE_BUTTON_LEFT&&' : '',
 						clickFn: this._.getClick(),
 						title: CKEDITOR.tools.htmlEncodeAttr( title || value ),
-						text: html || value
+						text: html || value,
+						language: language || editorLanguage
```

Comment:
If we go with the "add the `[lang]` attribute only if it's known" approach, then we could add the whole markup here (similar to [what we do in the `button` plugin](https://github.com/ckeditor/ckeditor4/blob/master/plugins/button/plugin.js#L316)).